### ISSUE TITLE
Enhancement: Turn on solution checking for external levels

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#141]
By default it would be nice to check the solutions of any external levels (from github/files).

**Comment by andybalaam** [comment]
It would be great to give visual feedback that the level is soluble.

**Comment by tttppp** [comment]
I started some work on this here:
https://github.com/tttppp/rabbit-escape/tree/verifyGithubSolutions?files=1

Done: Invalid github solutions cause exception.

Not done: Enable playing github levels with invalid solutions; Dialog with solution information; "There is a WON solution" feedback.

**Comment by andybalaam** [comment]
Cool, this is going to be very useful.

**Comment by andybalaam** [comment]
@tttppp did this ever get any further?

**Comment by tttppp** [comment]
No, I think I stopped work on this because the solution running code was being refactored. When I just tried merging with the latest master then there weren't many conflicts though.
